pdoc/doc.py

```diff
@@ -319,13 +322,22 @@ def members(self) -> dict[str, Doc]:
                     qualname,
                     docstring="",
                     annotation=self._var_annotations.get(name, empty),
-                    default_value=obj,
+                    default_value=_pydantic.default_value(self.obj, name, obj),
                     taken_from=taken_from,
                 )
-            if self._var_docstrings.get(name):
-                doc.docstring = self._var_docstrings[name]
-            if self._func_docstrings.get(name) and not doc.docstring:
-                doc.docstring = self._func_docstrings[name]
+
+            _docstring: str | None = None
+            if self.kind == "class":
+                _docstring = _pydantic.get_field_docstring(cast(type, self.obj), name)
+
+            if _docstring is None:
+                if self._var_docstrings.get(name):
+                    doc.docstring = self._var_docstrings[name]
+                if self._func_docstrings.get(name) and not doc.docstring:
+                    doc.docstring = self._func_docstrings[name]
+            else:
+                doc.docstring = _docstring
```

Comment:
Can we simplify like this?

```suggestion
            if doc := _pydantic.get_field_docstring(self.obj, name):
                doc.docstring = doc
            elif doc := self._var_docstrings.get(name):
                doc.docstring = doc
            elif doc := self._func_docstrings.get(name) and not doc.docstring:
                doc.docstring = doc
```